mephisto/abstractions/architects/router/flask/mephisto_flask_blueprint.py

```diff
@@ -35,6 +35,8 @@
 
 # Constants
 
+CURR_MEPHISTO_TASK_VERSION = "2.0.0"
```

Comment:
So it seems that this value is hardcoded in the file and then replaced when [scripts/sync_mephisto_task.py](https://github.com/facebookresearch/Mephisto/pull/675/files#diff-33575a8925bdd74eb5beca27cea0b35f16a38b02f857e253dd575225b13f47ff)

My initial expectation was that we would have the version lookup happen at run-time, e.g. when running `python run_task.py`? Any downsides to that? That approach would make sure that the number is "fresh" at all times.

If we do stick with this approach: how can we ensure that this value doesn't go stale, e.g. someone forgets to run the script when updating code? should we add a Github Action or pre-commit step to run  `sync_mephisto_task.py`?

Reply:
I think one issue with at runtime is if someone is using the PyPI package, they may not have the local dir and package structure to even be loading the `mephisto-task` package from, so I wanted to ensure consistency. I think having a precommit step for `sync_mephisto_task.py` would be sufficient. Do you have a pointer for adding that?

Reply:
PR now available at #676